doc/developer/workflow.rst

```diff
@@ -108,6 +108,34 @@ March/July/November.  Walking backwards from this date:
    (The :file:`configure.ac` edit and tag push are considered git housekeeping
    and are pushed directly to ``master``, not through a PR.)
 
+   Below is the snippet of the commands to use in this step.
+
+     .. code-block:: console
+
+        % git remote --verbose
+        origin    git@github.com:ton31337/frr (fetch)
+        origin    git@github.com:ton31337/frr (push)
+        upstream  https://github.com/frrouting/frr (fetch)
+        upstream  https://github.com/frrouting/frr (push)
+
+        % git checkout master
+        % git pull upstream master
+        % git checkout -b dev/8.2
+        % git tag base_8.2
+        % git tag frr-8.2-rc1
+        % git push upstream dev/8.2
+        % git checkout master
+        % sed -i 's/8.2-dev/8.3-dev/' configure.ac
+        % git tag -a frr-8.3-dev -m "frr-8.3-dev"
+        % git add configure.ac
+        % git commit -s -m "build: FRR 8.3 development version"
+        % git push upstream master
+
+   In this step, we also have to update packaging changelogs to reflect
+   the development version. More information is described in
+   :ref:`frr-release-procedure`. Changelogs need to be updated using
+   a standard way of development (Pull Requests) based on master branch.
+
```

Comment:
we should update the package version as well in the same step. Only change the  version number with no other changes. This will produce packages with the new version number that is height than any  previous version. Once the release is done, whatever  changes we make the changelog/packaging need to be cherry picked from the release  branch back to master.

Reply:
Added.